static/app/components/layouts/thirds.tsx

```diff
@@ -142,10 +142,43 @@ export const Body = styled('div')<{noRowGap?: boolean}>`
   }
 `;
 
+interface MainProps extends React.ComponentProps<typeof MainSection> {
+  children: React.ReactNode;
+  /**
+   * Set fullWidth to true to the layout does not have a side column.
+   * Defaults to false.
+   */
+  fullWidth?: boolean;
+  /**
+   * By default, the main content will span the width of the container.
+   * If set to 'constrained', the content will be wrapped in a 1440px wide container.
+   */
+  maxWidth?: 'constrained' | 'none';
```

Comment:
Could both of these be consolidated into `width="constrained | full"` with a default to `full`?